src/main/java/io/fabric8/elasticsearch/plugin/OpenShiftElasticSearchService.java

```diff
@@ -65,31 +66,41 @@ protected OpenShiftElasticSearchService(final Settings settings, final Client es
 		this.cache = cache;
 		this.aclNotifier = aclNotifer;
 		restController.registerFilter(aclFilter);
-		restController.registerFilter(new KibanaUserReindexFilter(settings, logger));
+		
+		boolean reindexEnabled = settings.getAsBoolean(OPENSHIFT_KIBANA_REWRITE_ENABLED_FLAG, OPENSHIFT_KIBANA_REWRITE_ENABLED_DEFAULT);
+		
+		if ( reindexEnabled ) {
+			restController.registerFilter(new KibanaUserReindexFilter(settings, logger));
+		}
 	}
 
 	@Override
 	protected void doStart() throws ElasticsearchException {
+		
+		boolean dynamicEnabled = settings.getAsBoolean(OPENSHIFT_DYNAMIC_ENABLED_FLAG, OPENSHIFT_DYNAMIC_ENABLED_DEFAULT);
+
```

Comment:
We are still letting users seed their initial ACL even if they disable the dynamic ACL.  From what I tried there wasn't a way to trigger that logic with the notifier disabled too.

I can add a debug statement if features are disabled/enabled.